templatefunc.go

```diff
@@ -470,6 +475,15 @@ func parseFormTag(fieldT reflect.StructField) (label, name, fType string, id str
 	id = fieldT.Tag.Get("id")
 	class = fieldT.Tag.Get("class")
 
+	meta := strings.Split(fieldT.Tag.Get("meta"), ",")
```

Comment:
I changed this, also I covered some edge cases. Please review now :) Thank you for considering this change